src/client/swagger/managedFunctions.yml

```diff
@@ -193,31 +219,72 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-    post:
-      operationId: PostFunctionsIDRuns
+  '/api/v2/functions/{functionID}/invoke':
+    get:
+      operationId: GetFunctionsIDInvoke
       tags:
         - Functions
-      summary: Manually trigger a function run
+      summary: Manually invoke a function with query params
       parameters:
         - in: path
           name: functionID
           schema:
             type: string
           required: true
+        - in: query
+          name: params
+          schema:
+            type: object
+            additionalProperties: true
+          style: form
+          explode: true
```

Comment:
As per the swagger 3.0 spec, this should evaluate to describe that query can be any key=value pair according to [this post on stack overflow](https://stackoverflow.com/questions/38187187/use-object-type-query-param-in-swagger-documentation/51277233#51277233) 
ie `/api/v2/functions/{functionID}/invoke?key1=value1&key2=value2` should be valid. 
but currently oats ignores `additionalProperties:true` and interprets this as:
ie `/api/v2/functions/{functionID}/invoke?params={key1=value1&key2=value2}` (or something like that)
oats needs to be updated to interpret this properly.

Reply:
https://github.com/influxdata/idpe/issues/9908